sycl/doc/developer/ContributeToDPCPP.md

```diff
@@ -62,6 +62,10 @@ end-to-end or SYCL-CTS tests.
 - Add a helpful comment describing what the test does at the beginning and
   other comments throughout the test as necessary.
 
+- All identifiers used in `llvm/sycl` headers and source files must contain at least one lowercase letter due to avoid conflicts with user-defined macros. 
+
+- Line length must not exceed 80 symbols.
+
```

Comment:
```suggestion
- All identifiers used in `llvm/sycl` headers and source files must contain at
  least one lowercase letter due to avoid conflicts with user-defined macros.
```